subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -442,73 +442,68 @@ static void wifi_mgmt_event_handler(struct net_mgmt_event_callback *cb,
 	}
 }
 
-static int __wifi_args_to_params(size_t argc, char *argv[],
+static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv[],
 				 struct wifi_connect_req_params *params,
 				 enum wifi_iface_mode iface_mode)
 {
 	char *endptr;
 	int idx = 1;
-	const struct shell *sh = context.sh;
+	//const struct shell *sh = context.sh;
```

Comment:
This can probably be removed if it is not needed?

Reply:
Done